features/shipping/managing_shipping_methods/sorting_shipping_methods_by_name_or_code.feature

```diff
@@ -16,15 +16,17 @@ Feature: Sorting listed shipping methods
         And I am logged in as an administrator
 
     @ui
-    Scenario: Shipping methods are sorted by code in ascending order by default
-        When I am browsing shipping methods
+    Scenario: Shipping methods can be sorted by code in ascending order
+        Given I am browsing shipping methods
+        When I start sorting shipping methods by code
```

Comment:
It's used both here and in products, I'd rather rename it in another PR as it is not a subject of this one.